tools/ansible/roles/dockerfile/templates/supervisor.conf.j2

```diff
@@ -22,12 +22,11 @@ stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
 
 [program:uwsgi]
-
 {% if kube_dev | bool %}
 command = make uwsgi
 directory = /awx_devel
 environment =
-    DEV_RELOAD_COMMAND='supervisorctl -c /etc/supervisord_task.conf restart all; supervisorctl restart tower-processes:daphne tower-processes:wsbroadcast'
+    DEV_RELOAD_COMMAND='supervisorctl -c /etc/supervisord_task.conf restart all; supervisorctl restart tower-processes:daphne'
```

Comment:
this is incorrect

Reply:
> it should not try to read the supervisor conf for the task container to restart daphne?

It is not. There are two separate commands in that line separated by a `;`